cfgov/ask_cfpb/tests/test_blocks.py

```diff
@@ -19,13 +19,13 @@ def setUp(self):
             '<div class="rich-text">Tip content</div>'
             '</aside>'
         )
+        self.expected_text_html = '<div class="rich-text">text</div>'
```

Comment:
For a cleaner git history, could we move this back to line 28?

Reply:
I will perform a squash and merge for a clean git history